src/search/pdbs/pattern_collection_generator_multiple_random.cc

```diff
@@ -0,0 +1,91 @@
+#include "pattern_collection_generator_multiple_random.h"
+#include "pattern_database.h"
+#include "random_pattern.h"
+#include "utils.h"
+
+#include "../option_parser.h"
+#include "../plugin.h"
+#include "../task_proxy.h"
+
+#include "../task_utils/causal_graph.h"
```

Comment:
Do we still need all of those includes? looks like most should now be included in the base class (probbaly the same for cegar).